extensions/wikia/EmbeddableDiscussions/scripts/ext.embeddableDiscussions.js

```diff
@@ -59,22 +59,25 @@ require([
 		});
 	}
 
-	function processData(threads, baseUrl, upvoteUrl) {
+	function processData(threads, upvoteUrl) {
 		var ret = [],
 			i,
 			thread,
-			userData;
+			userData,
+			date;
 
 		for (i in threads) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Restrict what this loop acts on by testing each property. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AForIn)